src/lightning/pytorch/core/module.py

```diff
@@ -1419,6 +1439,96 @@ def to_torchscript(
 
         return torchscript_module
 
+    @classmethod
+    def load_from_checkpoint(
```

Comment:
ModelIO is removed, all the code was put here. The only class that uses it

Reply:
Since it was public api, needs to be documented in changelog

Reply:
Good call. Done